deploy/olm-catalog/unifiedpush-operator/unifiedpush-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
+- currentCSV: unifiedpush-operator.v0.3.0
+  name: stable
```

Comment:
@KevFan @matskiv is there a need to have an 'integreatly' channel in here also?

Reply:
I don't think so. We can have that in our copy in integr8ly/manifests repo.